homeassistant/components/ruckus_unleashed/manifest.json

```diff
@@ -1,11 +1,11 @@
 {
   "domain": "ruckus_unleashed",
   "name": "Ruckus Unleashed",
-  "codeowners": ["@gabe565", "@lanrat"],
+  "codeowners": ["@lanrat", "@ms264556", "@gabe565"],
```

Comment:
Who's ms264556? Normally we want the user who wants to become a codeowner to open their own PR to be added.